package chapter17.task17_2;

import java.io.IOException;

public class Main {
	public static void main(String[] args) throws IOException {
		System.out.println("プログラムが起動します");
		throw new IOException();
	}

}
